modules/tectonic/resources/tectonic.sh

```diff
@@ -1,30 +1,20 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
I'm fine with Bash when we need its extensions, but I'd rather stick to the POSIX shell where possible.  Is this just for your `<<<` here-strings?

Reply:
Yep.